test/unit/presenters/formats/generic_edition_presenter_test.rb

```diff
@@ -5,10 +5,12 @@ class GenericEditionPresenterTest < ActiveSupport::TestCase
     setup do
       artefact = FactoryBot.create(:artefact)
 
-      expected_external_related_links = [
+      links = [
         { title: "GOVUK", url: "https://www.gov.uk" },
         { title: "GOVUK", url: "https://www.gov.uk" },
       ]
+      expected_external_related_links = ArtefactExternalLink.build(links)
+
       artefact.external_links = expected_external_related_links
```

Comment:
We could inline this variable (although `links` should be `expected_external_related_links ` if taking the previous comment into account).
```suggestion
      artefact.external_links = ArtefactExternalLink.build(links)
```